rest_helpers/responses.py

```diff
@@ -235,9 +235,9 @@ def _parse_filter(segment):
     return lambda x: compare(str(reduce(lambda acc,cur: acc[cur], path, x)), compared_to)
 
 
-def _response_from_error(framework_adapter, error):
+def _response_from_error(framework_adapter, error, header_dict={}):
```

Comment:
pass none instead of an empty dict. Passing an empty dict as a default value is dangerous because modifying the dict will modify the default value : 
```
>>> def my(x={}):
...    print(str(x))
...    x["a"]=1
...
>>> my
<function my at 0x101accf28>
>>> my()
{}
>>> my()
{'a': 1}
>>>
```

Reply:
also I suggest you name the parameter `headers`. It is usually expected to be a dict